types/index.d.ts

```diff
@@ -1,39 +1,14 @@
-// Type definitions for Apache Cordova InAppBrowser plugin
+// Type definitions for Apache Cordova InAppBrowser plugin 3.0
 // Project: https://github.com/apache/cordova-plugin-inappbrowser
 // Definitions by: Microsoft Open Technologies Inc <http://msopentech.com>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// 
+//
 // Copyright (c) Microsoft Open Technologies Inc
 // Licensed under the MIT license.
+// TypeScript Version: 2.3
+type channel = "loadstart" | "loadstop" | "loaderror" | "exit";
 
 interface Window {
-    /**
-     * Opens a URL in a new InAppBrowser instance, the current browser instance, or the system browser.
```

Comment:
all the `open` not needed any more?

Reply:
Below is a generic open method (`     open(url: string, target?: string, options?: string, replace?: boolean): InAppBrowser;
`) - this is just redundant when you have typings like target could be `"_self" | "_blank" | "_system" | string`.